### PR TITLE
Added warning for 'invalid' forms

### PR DESCRIPTION
**Pull request by iyves**
If the backend responds with 'invalid form' due to textract not being able to recognize or parse the uploaded file, a warning will be displayed at the top of the form sheet.

### CODE REVIEW

File: PWA/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-VUE_APP_SERVER_URL=https://beta-iddappserver.azurewebsites.net/
```
Comment (unstable-compound) [inline PWA/.env:1]: this should be blocked in the .gitignore, maybe you dont have the most recent version yet? Just remove this file from the commit, or you can solve it with any other merge conflicts

**Review by unstable-compound — CHANGES_REQUESTED** [review]
Aside from the .env file I give my approval, with the caveat that I only gazed through vue-specific things with an eye for style, not logic

File: PWA/.env (hunk 1 above)
Comment (keatonkraiger) [reply PWA/.env:1]: besides from this, I think it looks good, I have been testing it for a while

**Review by unstable-compound — APPROVED** [review]

**Review by keatonkraiger — APPROVED** [review]
I tested this thicc branch as well as I could and I think it has all tickets covered

**Pull request merged**